client/lib/react-helpers/index.js

```diff
@@ -11,6 +11,12 @@ export default {
 				? document.getElementById( domContainer )
 				: domContainer;
 
+		if ( typeof reduxStore.getState !== 'function'
```

Comment:
was this just a sanity check? were you having problems with an invalid store being passed?